rfd/0156-notifications.md

```diff
@@ -131,24 +149,30 @@ These will be stored under `/notification/global/<notification id>`.
 ```protobuf
 // GlobalNotification represents a global notification.
 message GlobalNotification {
+   // kind is the resource kind ("user_notification_state").
+   string kind = 1;
+   // sub_kind is the optional resource subkind. This is unused.
+   string sub_kind = 2;
```

Comment:
I wonder if it makes sense to use `sub_kind` for the type of notification, and get rid of the separate "type" field?